hingeto/hingeto.github.io
# Ch/add tac

**Overview** 

Changes in this PR affects the following 

1. Favour hyphen over underscore for url name for pages with compound words 
2. Adds link and pages for 
  1. FAQ
  2. Terms and Conditions
  3. Privacy Policy
3. Display display cookie consent message (EU regulation)
4. Add GitHub PR template

**Screenshots**
<img width="880" alt="screen shot 2018-08-23 at 11 12 07 pm" src="https://user-images.githubusercontent.com/17008072/44555445-20c5c900-a72d-11e8-99ef-70e9e0f97c2d.png">


**Steps to test**

1. Clone this branch
2. Run `$ jekyll serve`
3. Navigate to the running port on your browser to check it out
